test/unit/math/prim/mat/prob/categorical_logit_test.cpp

```diff
@@ -79,3 +81,58 @@ TEST(ProbDistributionsCategoricalLogit, error) {
   ns[1] = 12;
   EXPECT_THROW(categorical_logit_log(ns, theta), std::domain_error);
 }
+
+TEST(ProbDistributionsCategoricalLogit, error_check) {
+  using stan::math::categorical_logit_rng;
+  boost::random::mt19937 rng;
+
+  Matrix<double, Dynamic, 1> beta(3);
+
+  beta << 1.0, 10.0, -10.0;
+
+  EXPECT_NO_THROW(categorical_logit_rng(beta, rng));
+
+  beta << -1e3, 1.1e3, 1e5;
```

Comment:
Yeah, I considered deleting it cause it really doesn't test anything other than softmax's ability to handle absurd numbers. But I think in terms of the types of stuff that might get fed into a categorical_logit, no error is probly what folks would expect.